### ISSUE TITLE
Not working in VS2019 with SQLite 1.0.114.0?

### ISSUE THREAD

**Issue by tardezyx** [ErikEJ/SqlCeToolbox#904]
I have installed v4.8.728 of your tool on Win 10 LTSC x64 with VS2019 Enterprise v16.3.9.

Then I created a new C# winforms project in which I manually included the following SQLite references from the sqlite-netFx451-binary-bundle-x64-2013-1.0.114.0:
- SQLite.Designer.dll
- System.Data.SQLite.dll
- System.Data.SQLite.EF6.dll
- System.Data.SQLite.Linq.dll

Then I have included the following SQLite files manually within the project and set as "copy if newer":
- x86/SQLite.Interop.dll (from sqlite-netFx451-static-binary-Win32-2013-1.0.114.0)
- x64/SQLite.Interop.dll (from sqlite-netFx451-static-binary-x64-2013-1.0.114.0)
- System.Data.SQLite.dll (from sqlite-netFx451-static-binary-x64-2013-1.0.114.0)

Then I have added an existing SQLite DB-file from another project (in which the file already works) into the new project and tried out to add a new data source type using your tool which fails for the following variant:
- Add data source -> DataBase -> DataSet -> New connection
- Selected "SQLite Provider (Simple for EF6 by ErikEJ)"
- ConnectionString="Data Source=local.db;Version=3;"
- Test connection: successful
- OK => ERROR: The requested .Net Framework data provider cannot be found. It may not be installed.

I've also tried to use "SQL Server Compact 4.0 (Simple by ErikEJ)" which results in the same error message.

What is wrong/missing?

**Comment by ErikEJ** [comment]
Did you read the guide? https://github.com/ErikEJ/SqlCeToolbox/wiki/EF6-workflow-with-SQLite-DDEX-provider

**Comment by tardezyx** [comment]
I've now additionally downloaded and installed https://system.data.sqlite.org/downloads/1.0.114.0/sqlite-netFx46-setup-bundle-x64-2015-1.0.114.0.exe.

Checked for SQLite ED6 DbProvider in GAC in VS2019: It says "No". Closed VS2019.

Checked for the <system.data> <DbProviderFactories> ... entry in the machine.config-file. It was empty. I manually inserted the code from the "guide" and started VS2019 again. It still says "No".

So, I am still stuck.

As I am not sure what the purpose of all this installation mess is good for in the end, I will ask directly:

Does you tool give any benefit over SQLite Studio? E.g. is it possible to use a C# TableAdapter with your tool?
